doc/next/_sources/campaign-restrictions.rst.txt

```diff
@@ -0,0 +1,8 @@
+.. _campaign_restrictions:
+
+.. note::
+    **At the moment, the following restrictions apply to campaigns:**
+
+     - each seller account can have **at most one campaign**. In the next stage we plan to lift this restriction and bring the full potential of campaigns to everyone.
```

Comment:
Lets leave the "we plan to lift" as its unsure when this is, we don't want to make unknown promises

Reply:
I thought stages would also be explained to them during the first heads-up session. Or am I again in dreamland where nothing really happens :P